index.html

```diff
@@ -9,183 +9,168 @@
-
-      <p>Most of the samples use <a href="https://github.com/webrtc/adapter">adapter.js</a>, a shim to insulate apps from spec changes and prefix differences. (In fact, the standards and protocols used for WebRTC implementations are highly stable, and there are only a few prefixed names. For full interop information, see <a href="http://www.webrtc.org/web-apis/interop">webrtc.org/web-apis/interop</a>.)</p>
-
-      <p>In Chrome and Opera, all samples that use <code>getUserMedia()</code> must be run from a server. Calling <code>getUserMedia()</code> from a file:// URL will work in Firefox, but fail silently in Chrome and Opera.</p>
-
-      <p><a href="http://www.webrtc.org/testing" title="Command-line flags for WebRTC testing">webrtc.org/testing</a> lists command line flags useful for development and testing with Chrome.</p>
+        <p>
+            This is a collection of small samples demonstrating various parts of the <a href="https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API">WebRTC APIs</a>. The code for all
+            samples are available in the <a href="https://github.com/webrtc/samples">GitHub repository</a>.
+        </p>
 
-      <p>For more information about WebRTC, we maintain a list of resources at <a href="https://g.co/webrtc">g.co/webrtc</a>. If you've never worked with WebRTC, we recommend you start with the 2013 Google I/O <a href="https://www.youtube.com/watch?v=p2HzZkd2A40">WebRTC presentation</a>.</p>
+        <p>Most of the samples use <a href="https://github.com/webrtc/adapter">adapter.js</a>, a shim to insulate apps
+            from spec changes and prefix differences.</p>
 
-      <p>Patches and issues welcome! See <a href="https://github.com/webrtc/samples/blob/gh-pages/CONTRIBUTING.md">CONTRIBUTING.md</a> for instructions. The <a href="https://bit.ly/webrtcdevguide">Developer's Guide</a> for this repo has more information about code style, structure and validation. Head over to <a href="test/README.md">test/README.md</a> and get started developing.</p>
+        <p><a href="http://www.webrtc.org/testing" title="Command-line flags for WebRTC testing">https://webrtc.org/testing</a>
+            lists command line flags useful for development and testing with Chrome.</p>
 
-      <p class="warning"><strong>Warning:</strong> some of these demos will result in loud feedback if used without headphones.</p>
+        <p>Patches and issues welcome! See <a href="https://github.com/webrtc/samples/blob/gh-pages/CONTRIBUTING.md">CONTRIBUTING.md</a>
+            for instructions.</p>
 
+        <p class="warning"><strong>Warning:</strong> It is highly recommended to use headphones when testing these
+            samples, as it will otherwise risk loud audio feedback on your system.</p>
     </section>
 
     <section>
 
-      <h2 id="the-demos">The demos</h2>
+        <h2 id="getusermedia"><a href="https://developer.mozilla.org/en-US/docs/Web/API/Navigator/getUserMedia">getUserMedia()</a>
```

Comment:
A few nitpicky suggestions...

• Title in bold followed by explanation in normal weight.
• Use a colon, not a hyphen.
• Use the imperative verb: 'Access media devices' not 'Accessing media devices'.

So, something like this:

**[getUserMedia](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/getUserMedia)**: Access media devices